buildSrc/build.gradle.kts

```diff
@@ -1,9 +1,12 @@
-import org.gradle.kotlin.dsl.*
-
 plugins {
     `kotlin-dsl`
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
missclicked :crying_cat_face: : 

actually i want to downgrade only `api` module, not whole the project, because there are too mane dependencies on jfix-std-lib (which uses java-11 and depends only on `api` module in dynamic-property)

Reply:
now i tested my changes by  `gradle clean build publishToMavenLocal` command and checked published java-version in `api` module:
```
    "attributes": {
        "org.gradle.category": "library",
        "org.gradle.dependency.bundling": "external",
        "org.gradle.jvm.version": 8,
        "org.gradle.libraryelements": "jar",
        "org.gradle.usage": "java-api",
        "org.jetbrains.kotlin.localToProject": "public",
        "org.jetbrains.kotlin.platform.type": "jvm"
    }
```